spec/lib/appsignal/transaction_spec.rb

```diff
@@ -345,6 +345,111 @@
           expect(duplicate_transaction.is_duplicate).to be(true)
         end
       end
+
+      it "merges metadata from from the original transaction in the duplicate transaction" do
```

Comment:
It would be nice to test overriding in addition to merging. I wrote a test for that, feel free to throw it in the PR: https://github.com/appsignal/appsignal-ruby/commit/2dd1d0e10168a35c90ad6d16dcceacbb52f8cea7

(it was going to be a suggestion but it turned out too long)

Reply:
Thanks! I merged it 👍